doc/documentation.rst

```diff
@@ -9,7 +9,7 @@ Documentation
    :Release: |version|
    :Date: |today|
 
-Previous versions:  `0.12.0 <http://nipype.readthedocs.io/en/0.12.0/>`_ `0.11.0 <http://nipy.org/nipype/0.11.0>`_
+Previous versions:  `0.12.1 <http://nipype.readthedocs.io/en/0.12.1/>`_ `0.11.0 <http://nipy.org/nipype/0.11.0>`_
```

Comment:
This is intentionally dropping 0.12.0 from the list (rather than prepending 0.12.1)?